framework/sandstone.cpp

```diff
@@ -158,6 +158,7 @@ enum {
     test_delay_option,
     test_index_range_option,
     test_list_file_option,
+    test_builtin_list_option,
```

Comment:
NItpick: sort order

Reply:
yes, you do ;)
Neither enum/array, nor cases are sorted. These are kept together according the features, I kept this pattern.